packages/grafana-data/src/datetime/datemath.ts

```diff
@@ -102,11 +102,14 @@ export function parseDateMath(mathString: string, time: any, roundUp?: boolean):
   let i = 0;
   const len = strippedMathString.length;
 
+  const fy_start_month = 2;
```

Comment:
Why 2 not 1? Is it common to have February as start of the fiscal year? Thought it's specific to Grafana

Reply:
This is just a dummy variable I'm using to until we've figured out what we want to do about the config. This should be passed into datemaths somehow.

Reply:
I see, thanks!